### DISCUSSION TITLE
DS920+ SSD to HDD transfer speed

### DISCUSSION THREAD

**Discussion by jhlchu** [007revad/Synology_HDD_db#380] [General]
I have a single SSD setup as a place where a docker container downloads stuff, then I move them into the HDD SHR pool from the web interface (File Station). The thing is, the transfer speed only maxes out at 85MB/s. Is this just an issue with the old PCIE bus because I can get at least 113 in network file transfer from both the SSD and HDDs.

**Comment by 007revad** [comment]
Sorry. Somehow I forgot to answer this one.

The PCIe 2.0 bus is capable of a lot more than 85MB/s

What read and write speeds do these commands return?
```
sudo hdparm -tT --direct /dev/sata1
sudo hdparm -tT --direct /dev/sata2
sudo hdparm -tT --direct /dev/sata3
sudo hdparm -tT --direct /dev/sata4
```

Synology's apps, including File Station are slow. They probably limit their speed to prevent making the NAS too slow for other uses at the same time.

**Comment by jhlchu** [comment]
```
/dev/sata1:
 Timing O_DIRECT cached reads:   914 MB in  2.00 seconds = 456.78 MB/sec
 Timing O_DIRECT disk reads: 684 MB in  3.00 seconds = 227.90 MB/sec

/dev/sata2:
 Timing O_DIRECT cached reads:   950 MB in  2.00 seconds = 474.43 MB/sec
 Timing O_DIRECT disk reads: 722 MB in  3.00 seconds = 240.34 MB/sec

/dev/sata3:
 Timing O_DIRECT cached reads:   976 MB in  2.00 seconds = 488.01 MB/sec
 Timing O_DIRECT disk reads: 810 MB in  3.00 seconds = 269.97 MB/sec

/dev/sata4:
 Timing O_DIRECT cached reads:   966 MB in  2.00 seconds = 482.11 MB/sec
 Timing O_DIRECT disk reads: 762 MB in  3.00 seconds = 253.85 MB/sec
 ```
I guess it would be faster to move it with Putty directly then.

**Reply by 007revad** [reply]
> I guess it would be faster to move it with Putty directly then.

Yes

**Reply by jhlchu** [reply]
Huh, weird result. I used `rsync` to move the files and I'm getting 65-73 MB/s, much slower than just using move in Windows explorer.

**Reply by 007revad** [reply]
That is weird.
